src/app/ReadHandler.h

```diff
@@ -280,7 +280,13 @@ class ReadHandler : public Messaging::ExchangeDelegate
     // are waiting for the max reporting interval to elaps.  When mHoldSync
     // becomes false, we are allowed to send an empty report to keep the
     // subscription alive on the client.
-    bool mHoldSync                   = false;
+    bool mHoldSync = false;
+    // For subscriptions, we record the timestamp when we started to generate the last report.
+    // The mCurrentReportTick records the timestamp for the current report, which won;t be used for checking if this
+    // ReadHandler is dirty.
+    // mLastReportTick will be set to mCurrentReportTick after we sent the last chunk of the current report.
+    uint64_t mLastReportTick         = 0;
```

Comment:
The description isn't quite accurate - this tracks the timestamp of the first report in a train of reports that had last been sent to the client. This is updated at the end of a given report train.

As such, it should not be called `mLastReportTick` but rather, `mPreviousReportsBeginTime`.